functions/package.json

```diff
@@ -2,12 +2,19 @@
   "name": "functions",
   "description": "Cloud Functions for Firebase",
   "scripts": {
-    "lint": "eslint .",
-    "serve": "firebase emulators:start --only functions",
-    "shell": "firebase functions:shell",
+    "config:get": "npx firebase functions:config:get",
+    "config:test": "npx firebase use test",
+    "delete:poll": "npx firebase functions:delete poll",
+    "delete:poll:force": "npx firebase functions:delete poll --force",
+    "deploy": "npx firebase deploy --only functions",
+    "deploy:poll": "npx firebase deploy --only functions:poll",
+    "lint": "npx eslint .",
+    "lint:fix": "npx eslint . --fix",
+    "logs": "npx firebase functions:log",
+    "reboot:poll": "run-s delete:poll:force deploy:poll",
+    "serve": "npx firebase emulators:start --only functions",
+    "shell": "npx firebase functions:shell",
```

Comment:
changes:
- `npx` used to future-proof `devDependencies` add (see #33)
- `npm-run-all` used for flexible scripts (also`devDependencies`)
- common `firebase-cli` use cases included because otherwise we're "thinking in firebase-cli"